### DISCUSSION TITLE
Import meeting error message "Either Address or City is required."

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1182] [Q&A]
There are new online meetings I want to upload to the website. 

I've downloaded the example spreadsheet and copleted it with the new meeting information, see the spreadsheet here (which i downloaded to csv, prior to uploading): 
https://docs.google.com/spreadsheets/d/1D5mNclnBQe-d0HUtIsFuaEKBNFBN5BT9J9rooQHcOF0/edit?usp=sharing 

As they are online meetings I have left the "Location" field blank, is this what causes the error message "Either Address or City is required."? 

I did add an address to a location in the suburb hosting an online meeting, so the meeting appears when members look for a meeting near them.

Do you have any advice?

**Comment by anchovie91471** [comment]
I've requested access to your spreadsheet so I can have a look.

**Reply by nataliefaith2** [reply]
Access granted

**Comment by anchovie91471** [comment]
Have you read our FAQ on this topic?

https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20use%20this%20plugin%20to%20list%20telephone%20meetings%20or%20other%20meetings%20without%20a%20fixed%20location%3F

**Reply by nataliefaith2** [reply]
Thanks for sharing these FAQs. I've added an address into the "address" field so on maps it is easily found by member searching within a region/district, I just haven't added anything in the "location" field. Is this needed to successfully import?
